### PR TITLE
feat: Add floating action button page

### LINKED ISSUE #24: Create FAB component page

### PR DESCRIPTION
**Pull request by moog16**
fixes #24 

NOTE: I edited the Landing page SVG. It had an extended FAB in the image, but we currently don't support that. I made the SVG look like it does in the Landing page mock: https://gallery.googleplex.com/projects/MCHbtQVoQ2HCZToS_wX_kMCN/files/MCHtA7U1iMGr6_KZPshJxIAhD0opuTKYyuk

### CODE REVIEW

File: config/webpack.config.dev.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const publicUrl = '';
 const env = getClientEnvironment(publicUrl);
 
 // List of all component names for generating the corresponding html pages.
-const components = ['button'];
+const components = ['button', 'fab'];
```
Comment (lynnjepsen) [inline config/webpack.config.dev.js]: new line!

otherwise we merge conflict constantly!

File: config/webpack.config.prod.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ const extractTextPluginOptions = shouldUseRelativeAssetPaths
   : {};
 
 // List of all component names for generating the corresponding html pages.
-const components = ['button'];
+const components = ['button', 'fab'];
```
Comment (lynnjepsen) [inline config/webpack.config.prod.js]: same here!

File: src/FabPage.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  render() {
+    return (
+      <div>
+        <h3 className='mdc-typography--subheading2'>Standard Floating Action</h3>
+        <Fab />
+
+        <h3 className='mdc-typography--subheading2'>Mini Floating Action</h3>
+        <Fab mini />
+
+        <h3 className='mdc-typography--subheading2'>Exiting Floating Action</h3>
```
Comment (lynnjepsen) [inline src/FabPage.js]: drop the "exiting floating action"....thats a demo. Its not necessary in a catalog

File: config/webpack.config.dev.js (hunk 1 above)
Comment (kfranqueiro) [reply config/webpack.config.dev.js]: Yeah, I thought that too... the problem is the damage has already been done with multiple pending PRs modifying these lines :(

**Pull request merged**